sdk/route/src/manual.ts

```diff
@@ -145,6 +148,7 @@ export class NttManualRoute<N extends Network>
         token: request.destination.id,
         amount: amount.parse(params.amount, request.destination.decimals),
       },
+      eta: finality.estimateFinalityTime(request.fromChain.chain),
```

Comment:
I just realized that if we only add `eta` to quotes, then when resuming a manual route the UI will have no way of counting down time or showing a progress bar (because the receipt doesn't have any ETA in it). this probably isn't a showstopper, just calling it out.

Reply:
Right now the `eta` is just an estimate for how long it takes for the attestation to become available (essentially the source chain's finality time)